pubspec.yaml

```diff
@@ -12,7 +12,7 @@ environment:
   sdk: '>=2.0.0 <3.0.0'
 
 dependencies:
-  vm_service_lib: ^0.3.5
+  vm_service: ^2.1.1
   watcher: ^0.9.7
   glob: ^1.1.3
```

Comment:
Maybe take the opportunity to upgrade to latest version (1.2.0)?

Reply:
Latest version of what?

Reply:
This line above :)

`glob: ^1.1.3`

Line 17, where this comment is!

Reply:
My screen shows an except of 6 lines, nothing highlighting line 17 specifically - so I think maybe GitHub has a bit of a UI problem there. 😉

Anyhow, the changes I made do not in any way relate to the use of `glob` - the package is still compatible with the `1.1` series of this package, as no new features from the `1.2` series are required, so, no.

You should let the consumer and/or package manager decide which version they need/want - this is the problem that package managers were intended to solve. The system works. 😄